.github/workflows/build.yml

```diff
@@ -192,3 +285,47 @@ jobs:
        with:
          name: ${{matrix.version}}-${{matrix.os}}-${{matrix.vm}}
          path: workspace/target/*
+  
+  windows_smokeTest:
+    name: windows_smokeTest
+    runs-on: windowss-latest
+    needs:
+      - build_windows
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [windows-2019]
+        version: [jdk]
+        vm: [hotspot]
+        include:
+          - os: windows-2016
+            version: jdk11u
+            vm: hotspot
+    steps:
+    - uses: actions/checkout@v2
+    - name: Restore build artifacts
+      uses: actions/download-artifact@v2
+      with:
+        name: ${{matrix.version}}-${{matrix.os}}-${{matrix.vm}}
+        path: ~/${{matrix.version}}-${{matrix.os}}-${{matrix.vm}}
+    - name: Unpack jdk
+      run: |
+        tar -xf "${HOME}/${{matrix.version}}-${{matrix.os}}-${{matrix.vm}}/OpenJDK.tar.gz" -C "${HOME}/${{matrix.version}}-${{matrix.os}}-${{matrix.vm}}"
```

Comment:
this is going to need to be an unzip command

Reply:
You likely need a rebase to pick up https://github.com/adoptium/temurin-build/commit/631f4e3d41b2c33fec83fad88a9264c8130b73d0